easybuild/easyconfigs/p/Python/Python-2.7.9-foss-2015.05.eb

```diff
@@ -0,0 +1,115 @@
+name = 'Python'
+version = '2.7.9'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'foss', 'version': '2015.05'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.9.1'
+scipyversion = '0.14.1'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+    ('SQLite', '3.8.8.1'),
```

Comment:
hi folks,

sorry for jumping so late in the action; since we're still in `develop` branch, doesn't this one worth an upgrade?
I'm basically reading this comment over here: https://www.sqlite.org/
```
Version 3.8.10.2 of SQLite is recommended for all new development. Upgrading from all prior releases is recommended.
```

The seem to recommend it twice as much :)